packages/edit-post/src/components/layout/index.native.js

```diff
@@ -143,8 +144,10 @@ class Layout extends Component {
 						parentHeight={ this.state.rootViewHeight }
 						style={ toolbarKeyboardAvoidingViewStyle }
 					>
-						{ showPageTemplatePicker && (
-							<__experimentalPageTemplatePicker />
+						{ isPage && (
```

Comment:
Before, the component was being unmounted after the user has at least one block. To be able to have a fade-out animation, the component can't be unmounted. So now the picker will be rendered if its a page only (since we don't want to import everything for a standard post)

Within the component, it will handle if it should be rendered or not by passing `showPageTemplatePicker` as a prop.